src/box/xrow_update_field.c

```diff
@@ -584,29 +592,38 @@ xrow_update_op_store_bit(struct xrow_update_op *op,
 	return end - out;
 }
 
+/**
+ * Copy string rope leaf data to the output buffer.
+ *
+ * Arguments:
+ *  data - leaf data pointer
+ *  size - leaf data size
+ *  cb_arg - pointer to the output buffer pointer
+ *
+ *  Buffer pointer updated to point after the copied data.
```

Comment:
If you want to describe the args, then use the doxygen format, please. Same in all the other new code.